libs/core-ui/src/lib/util/TimeUtils.ts

```diff
@@ -20,11 +20,18 @@ export function isTimeOrTimeSeriesIDColumn(
   featureName: string,
   featureMetaData?: IFeatureMetaData
 ): boolean {
-  if (featureMetaData) {
-    const isDatetimeFeature = featureMetaData.time_column_name === featureName;
+  // The if-condition below should always be true since these are all required
+  // for forecasting and validated upon building the model assessment context.
+  // The if statement exists merely to satisfy the compiler.
+  if (
+    featureMetaData &&
+    featureMetaData.datetime_features &&
+    featureMetaData.datetime_features?.length > 0
+  ) {

```

Comment:
can remove the question mark? 
(
    featureMetaData &&
    featureMetaData.datetime_features &&
    featureMetaData.datetime_features.length > 0
  )

Reply:
also: if(
featureMetaData?.datetime_features &&
featureMetaData.datetime_features.length > 0
)